hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
         """HiPS access url"""
-        return self.data['moc_access_url'].rsplit('/', 1)[0]
+        try:
+            return self.data['hips_service_url']
+        except IndexError:
+            return self.data['moc_access_url'].rsplit('/', 1)[0]
+        except:
```

Comment:
Is this the correct way to handle this?

```
    def base_url(self) -> str:
        """HiPS access url"""
        try:
            return self.data['hips_service_url']
        except KeyError:
            return self.data['moc_access_url'].rsplit('/', 1)[0]
        except KeyError:
            return self.data['properties_url']
        except:
            return ValueError('URL does not exist!')

```

I'm getting this warning in PyCharm:

> Exception class 'KeyError' has already been caught